native/packages/react-native-bpk-component-banner-alert/stories.js

```diff
@@ -117,13 +120,13 @@ class BpkBannerAlertFadeDemo extends React.Component {
         <BpkButton
           title="Add banner alert!"
           onPress={this.addBannerAlert}
-          style={styles.bannerAlert}
```

Comment:
So it this a breaking change or did `style` used to have no effect at all?

Reply:
`style` is still applied to the outer container so no breaking change here.

Reply:
Yup :+1:. Before all user styling was applied to the outer View. Now we're just providing an extra prop `bannerStyle` that will be applied to the banner View (ie not the animated view)